server/src/main/java/com/qubole/quark/serverclient/server/Main.java

```diff
@@ -0,0 +1,164 @@
+/*
```

Comment:
Be careful about copyright note ...if you have copied significant parts from any project ..make sure you retain the copyright note too. Don't alter that.